lib/Factory/TermFactory.php

```diff
@@ -66,6 +76,14 @@ protected function from_term_obj(object $obj) : CoreInterface {
 		));
 	}
 
+	protected function from_taxonomy_names(array $names) {
+		return $this->from_wp_term_query(new WP_Term_Query(
+			$this->filter_query_params([
+				'taxonomy' => $names,
+			])
+		));
+	}
+
 	protected function get_term_class(WP_Term $term) : string {
 		// Get the user-configured Class Map
 		$map = apply_filters( 'timber/term/classmap', [
```

Comment:
I see that in the documentation for [Term Class Maps](https://github.com/timber/timber/blob/2.x-docs-api/docs/guides/class-maps.md#the-term-class-map), we currently overwrite the passed Class Maps in the examples. I should probably update the documentation with `array_merge()`, like this:

```php
use Genre;

add_filter( 'timber/term/classmap', function( $classmap ) {
    return array_merge( $classmap, [
        'genre' => Genre::class,
    ] );
} );

```

Reply:
Agreed, I'll let you do that.

Reply:
Updated in https://github.com/timber/timber/pull/2073/commits/6db9964fd10164d65f5f7eb136d647d7d85fdcdd